1-js/02-first-steps/08-operators/article.md

```diff
@@ -106,6 +106,11 @@ alert(2 + 2 + '1' ); // "41" and not "221"
 
 Here, operators work one after another. The first `+` sums two numbers, so it returns `4`, then the next `+` adds the string `1` to it, so it's like `4 + '1' = 41`.
 
+```js run
+alert('1' + 2 + 2); // "122" and not "14"
+```
+Here, the first operand is a string, the compiller treats the other two operands as string too. The `2` gets concatinated to `'1'`, so it's like `'1' + 2 = "12"` and `"12" + 2 = "122"`
```

Comment:
Not sure, but maybe (correct possible typos and unify quotes):

```suggestion
Here, the first operand is a string, the compiler treats the other two operands as strings too. The `2` gets concatenated to `'1'`, so it's like `'1' + 2 = '12'` and `'12' + 2 = '122'`.
```

Reply:
For consistency in quotes with above paragraphs, it's better to change

```
`'1' + 2 = '12'` and `'12' + 2 = '122'`.
```
to 

```
`'1' + 2 = "12"` and `"12" + 2 = "122"`.
```

This is not a must, and more like a personal preference.

Reply:
Yes, that's why I put single quotes there in the initial commit, but a suggestion was made by a contributor, so, I thought he must be right because of more experience in this project.
Should I change those quotes now or not?

Reply:
Not so much experience, sorry) It was just a suggestion, so feel free to stick to your initial thought)